types/node/v10/util.d.ts

```diff
@@ -87,24 +87,27 @@ declare module "util" {
         fn: (arg1: T1, arg2: T2, arg3: T3, arg4: T4, arg5: T5, arg6: T6) => Promise<TResult>
     ): (arg1: T1, arg2: T2, arg3: T3, arg4: T4, arg5: T5, arg6: T6, callback: (err: NodeJS.ErrnoException | null, result: TResult) => void) => void;
 
+    type Errback = (err?: Error | null) => void;
```

Comment:
Can we perhaps rename this to `Callback` and the other one `ValueCallback`?
`Errback` seems like slang.

Reply:
Sure 👍